packages/editor-ui/src/components/NodeSettings.vue

```diff
@@ -28,11 +28,14 @@
 			</div>
 			<div v-if="isCommunityNode" :class="$style.descriptionContainer">
 				<div class="mb-l">
-					<span
-						v-html="$locale.baseText('nodeSettings.communityNodeUnknown.description', { interpolate: { packageName: node.type.split('.')[0] } })"
-						@click="onMissingNodeTextClick"
-					>
-					</span>
+					<i18n path="nodeSettings.communityNodeUnknown.description" tag="span" @click="onMissingNodeTextClick">
+						<template #action>
+							<a
+								:href="`https://www.npmjs.com/package/${node.type.split('.')[0]}`"
+								target="_blank"
+							>{{ node.type.split('.')[0] }}</a>
+						</template>
+					</i18n>
```

Comment:
Look like unknown community mode modal is not rendering correctly with these changes (it shows additional content after _how to install_ link):
![SCR-20220905-fgt](https://user-images.githubusercontent.com/2598782/188413637-3be5dcaf-5981-48ff-bd65-00ec8e09212f.png)